src/odemis/driver/andorcam3.py

```diff
@@ -831,10 +834,10 @@ def _getTargetTemperatureRange(self):
                 tmps = [float(t) for t in tmps_str if tmps_str is not None]
                 if not self.isWritable(u"TemperatureControl"):
                     # Still allow 25°C to disable the cooling
-                    return min(tmps), 25
+                    return min(tmps), TEMP_NO_COOLING
                 else:
                     # TODO: just return a set
-                    return min(tmps), max(tmps)
+                    return min(tmps), max(tmps + [TEMP_NO_COOLING])
```

Comment:
why is the TEMP_NO_COOLING added to tmps to get the maximum?

Reply:
Actually, we want to return  TEMP_NO_COOLING, but in case the maximum temperature is higher than this, we'd return it. 

It should really never happen because the TEMP_NO_COOLING was picked to be higher than any useful cooling temperature. So I've now simplified the code, and just log a warning if this very unlikely situation happens.